### DISCUSSION TITLE
media elements

### DISCUSSION THREAD

**Discussion by alecko85** [mjmlio/mjml#2550] [Q&A] [answered]
I'm trying to put some mj-social-element but it doesn't work like was one week ago. 

`<mj-social font-size="15px" icon-size="30px" mode="horizontal" padding="0" align="center">
            <mj-social-element name="facebook" href="some link" background-color="#A1A0A0">
            </mj-social-element>
            <mj-social-element name="linkedin" href="some link" background-color="#A1A0A0">
            </mj-social-element>
          </mj-social>`

**Comment by iRyusa** [comment, marked as answer]
Please check the issue tracker before opening a new one
https://github.com/mjmlio/mjml/issues/2547
